goal_src/jak2/engine/gfx/mood/mood-tables.gc

```diff
@@ -7,3 +7,867 @@
+                                           :amb-color (new 'static 'vector :x 0.3714 :y 0.3964 :z 0.4877 :w 1.0)
+                                           )
+                                         (new 'static 'mood-color
+                                           :lgt-color (new 'static 'vector :x 0.1616 :y 0.1579 :z 0.1486)
+                                           :amb-color (new 'static 'vector :x 0.3684 :y 0.4604 :z 0.5124 :w 1.0)
+                                           )
+                                         (new 'static 'mood-color
+                                           :lgt-color (new 'static 'vector :x 0.1841 :y 0.1802 :z 0.174)
+                                           :amb-color (new 'static 'vector :x 0.3684 :y 0.4604 :z 0.5124 :w 1.0)
+                                           )
+                                         (new 'static 'mood-color
+                                           :lgt-color (new 'static 'vector :x 0.1616 :y 0.1579 :z 0.1486)
+                                           :amb-color (new 'static 'vector :x 0.3684 :y 0.4604 :z 0.5124 :w 1.0)
+                                           )
+                                         (new 'static 'mood-color
+                                           :lgt-color (new 'static 'vector :x 0.1646 :y 0.1549 :z 0.1234)
+                                           :amb-color (new 'static 'vector :x 0.3648 :y 0.4145 :z 0.5142 :w 1.0)
+                                           )
+                                         (new 'static 'mood-color
+                                           :lgt-color (new 'static 'vector :x 0.0457 :y 0.0412 :z 0.0499)
+                                           :amb-color (new 'static 'vector :x 0.2951 :y 0.3406 :z 0.4329 :w 1.0)
+                                           )
+                                         (new 'static 'mood-color
+                                           :lgt-color (new 'static 'vector :x 0.0644 :y 0.0713 :z 0.0823)
+                                           :amb-color (new 'static 'vector :x 0.3593 :y 0.3592 :z 0.4941 :w 1.0)
+                                           )
+                                         (new 'static 'mood-color
+                                           :lgt-color (new 'static 'vector :x 0.0456 :y 0.0574 :z 0.0531)
+                                           :amb-color (new 'static 'vector :x 0.3186 :y 0.3784 :z 0.4382 :w 1.0)
+                                           )
+                                         )
+                                 )
+  )
+
+(define *no-cloud-mood-channel-group*
+  (new 'static 'mood-channel-group
+    :data (new 'static 'inline-array mood-channel 4
+      (new 'static 'mood-channel
+        :vecs (new 'static 'inline-array vector4 6
+          (new 'static 'vector4)
+          (new 'static 'vector4 :data (new 'static 'array float 4 0.0 0.2 0.4 1.0))
```

Comment:
Can you add `:score -1` to the `data` in `deftype vector4`? I think that will make these look a lot nicer, and I meant to do this originally.